lab03-kaylee/test/fs-textData-test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+const fileReader = require('../fs-textData.js');
+let mockArray = [ '646f677320617265', '6361747320617265', '6269726473206172' ];
+
+describe('fs-textData.js', function() {
+  describe('#readFile', function() {
+    it('should be an object', done => {
+      fs.readFile(`${__dirname}/../data/one.txt`, function(err, data) {
+        let bufferData = data;
+        expect(typeof bufferData).to.be.equal(typeof Buffer.from(''));
+        done();
+      });
+    });
+  });
+});
+
+describe('fs-textData.js', function() {
+  describe('#readFile', function() {
+    it('should be an object', done => {
+      fs.readFile(`${__dirname}/../data/two.txt`, function(err, data) {
+        let bufferData = data;
+        expect(typeof bufferData).to.be.equal(typeof Buffer.from(''));
+        done();
+      });
+    });
+  });
+});
+
+describe('fs-textData.js', function() {
+  describe('#readFile', function() {
+    it('should be an object', done => {
+      fs.readFile(`${__dirname}/../data/three.txt`, function(err, data) {
+        let bufferData = data;
+        expect(typeof bufferData).to.be.equal(typeof Buffer.from(''));
+        done();
+      });
+    });
+  });
+});
+
+describe.only('fs-textData.js', function() {
```

Comment:
No points lost for this, but remember to take off the ``.only`` before submitting since I was only able to see 1 test run and got confused. :)